alteryx/featuretools
# v0.22.0

**v0.22.0 Nov 30, 2020**
* Enhancements
    * Allow variable descriptions to be set directly on variable (#1207)
    * Add ability to add feature description captions to feature lineage graphs (#1212)
    * Add support for local tar file in read_entityset (#1228)
* Fixes
    * Updates to fix unit test errors from koalas 1.4 (#1230, #1232)
* Documentation Changes
    * Removed link to unused feedback board (#1220)
    * Update footer with Alteryx Innovation Labs (#1221)
    * Update links to repo in documentation to use alteryx org url (#1224)
* Testing Changes
    * Update release notes check to use new repo url (#1222)
    * Use new version of pull request Github Action (#1234)
    * Upgrade pip during featuretools[complete] test (#1236)
    * Migrated CI tests to github actions (#1226, #1237, #1239)
